bin/pycbc_live

```diff
@@ -241,15 +241,20 @@ class LiveEventManager(object):
 
             live_ifos = [ifo for ifo in fud if 'snr_series' in fud[ifo]]
 
+            time_offset = 0
+            rtoff = 'foreground/{}/time_offset'.format(coinc_ifos[0])
+            if rtoff in coinc_results:
+                time_offset = coinc_results[rtoff]
             event = SingleCoincForGraceDB(live_ifos, coinc_results, bank=bank,
                                           psds=psds, followup_data=fud,
                                           low_frequency_cutoff=f_low,
                                           channel_names=args.channel_name,
-                                          mc_area_args=self.mc_area_args)
+                                          mc_area_args=self.mc_area_args,
+                                          time_offset=time_offset)
 
             end_time = int(coinc_results['foreground/%s/end_time'
                                          % coinc_ifos[0]])
-            fname = 'coinc-{}-{}.xml.gz'.format(end_time, pycbc.random_string(6))
+            fname = 'coinc-{}-{}.xml.gz'.format(end_time + time_offset, pycbc.random_string(6))
```

Comment:
Ah right, we also need the time offset down here. However, I propose giving `SingleCoincForGraceDB` a property called `merger_time_estimate`. This would allow us to replace the above *two* lines with this single line:
```python
fname = 'coinc-{:.0f}-{}.xml.gz'.format(event.merger_time_estimate,
                                        pycbc.random_string(6))
```

Reply:
Ok, moving to a property of SingleCoincForGracedb and consolidating.